modules/UI/toolbars/ToolbarToggler.js

```diff
@@ -35,7 +35,6 @@ function hideToolbar(force) { // eslint-disable-line no-unused-vars
     toolbarTimeoutObject = null;
 
     if (Toolbar.isHovered()
-            || APP.UI.isRingOverlayVisible()
```

Comment:
Do we really want to do this because If I remember correctly the initial requirements for the ring overlay was to display the toolbar all the time?

Also currently if the ring overlay is displayed when the toolbar disappear there is no way to bring the toolbar back. Is this a bug?

I think it's important to be able to use the hangup button at least for some use cases which is impossible now.